w3af/core/ui/api/utils/digital_certificate.py

```diff
@@ -0,0 +1,49 @@
+from w3af.core.controllers.misc.homeDir import get_home_dir
+import os
+
+from OpenSSL import crypto
+import socket,ssl
+
+class SSLCertifiacate:
+    def __init__(self):
+        ssl_dir = os.path.join(get_home_dir(),"ssl")
```

Comment:
Please respect PEP-8 , after a ',' there should be a space.

Same applies to all the other pep-8 related stuff in your change.